docs/containerd-2.0.md

```diff
@@ -0,0 +1,105 @@
+
+## What's new
+
+### Adds top level configuration migration support
+
+This release adds containerd configuration migration support during daemon startup time. This feature means previous versions of containerd configuration will continue to work on future daemon versions; however, it is recommended to migrate to the latest version (`version = 2`) to avoid migrations and optimize daemon startup time. See [RELEASES.md](../RELEASES.md#daemon-configuration) for full details.
+
+### Adds deprecations warnings to Introspection API
+
+This release adds deprecations warnings to the `ServerResponse` for the introspection service. (`/containerd.services.introspection.v1.Introspection/Server`)
+
+To ease containerd 2.0 transition, the community has moved to backport this feature to containerd 1.6 and 1.7 release branches to give users another tool to identify the sticking points for migrating containerd versions.
+
+Administrators whose workloads are running on containerd versions >= 1.6.25, >= 1.7.9 can query their containerd server to receive  warnings for features marked for deprecation and are in the critical path. With the `ctr` client, users can run `ctr deprecations list`.
+
+### Adds plugin info to Introspection API
+
+This release adds the `PluginInfo` API definition to the introspection service. (`/containerd.services.introspection.v1.Introspection/PluginInfo`)
+
+`PluginInfo` defines the API for returning information directly from a [containerd] plugin [if the plugin supports it].
+
+### Adds Sandbox Controller Update API
+
+This release adds the `Update` API definition for the sandbox controller. (`/containerd.services.sandbox.v1.Controller/Update`)
+
+`Update` defines the API that sandbox plugins will implement to allow for modifications to an existing sandbox's attributes such as [sandbox] specification, runtime, extensions, and labels.
+
+### Adds OTEL environment variable configuration support for containerd's built-in tracing plugin
+
+This release adds support for the standard environment variables defined in the OpenTelemetry [specification](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/) to the built-in containerd tracing plugin.
+
+Implementation note: by default OTEL will try to connect to `https://localhost:4318` if no endpoint is set. In contrast, the containerd tracing OLTP plugin will be disabled if no endpoint is configured.
+
+i.e. both `OTEL_SDK_DISABLED` and one of either `OTEL_EXPORTER_OTLP_ENDPOINT` or `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` must be set in the containerd environment for the tracing plugin to be enabled.
+
+### Enables NRI by default
+
+This release enables NRI by default. To disallow NRI functionality, users will need to explicitly disable the common NRI plugin in containerd configuration. See [Disabling NRI Support in Containerd](../docs/NRI.md#disabling-nri-support-in-containerd) for details.
```

Comment:
We should describe what NRI is and why it's on by default rather than only discussing how it can be disabled.

Reply:
Reworked this to highlight NRI. As for why it is on, I struggled a little bit. IIUC NRI is safe for all use cases.